CONTRIBUTING.rst

```diff
@@ -59,12 +59,13 @@ Get Started!
 
 Ready to contribute? Here's how to set up `simpler_eda` for local development.
```

Comment:
Ready to contribute? 

In general, we ask you to follow the [GitHub Flow workflow](https://guides.github.com/introduction/flow/). 

Here's how to set up `simpler_eda` for local development.

Reply:
Got it!